Formula/erigon.rb

```diff
@@ -0,0 +1,49 @@
+class Erigon < Formula
+  desc "Implementation of Ethereum (execution client), on the efficiency frontier"
+  homepage "https://github.com/ledgerwatch/erigon"
+  url "https://github.com/ledgerwatch/erigon/archive/refs/tags/v2.31.0.tar.gz"
+  sha256 "eaed8d4e24d813991a53de0b8e89161ed5090168ddae8487e20be89dd861782d"
+  license "LGPL-3.0-or-later"
```

Comment:
```suggestion
  license all_of: ["GPL-3.0-or-later", "LGPL-3.0-or-later"]
```

license search, https://github.com/search?q=repo%3Aledgerwatch%2Ferigon+GENERAL+PUBLIC+LICENSE&type=code
For example, this [cmd/evm/internal/compiler/compiler.go](https://github.com/ledgerwatch/erigon/blob/ec582dd9b33279a51d619e6c66073581b6bb0b22/cmd/evm/internal/compiler/compiler.go#L1-L15) is using `GPL-3.0-or-later`